script.js

```diff
@@ -0,0 +1,182 @@
+    buttonOne: 'See Live',
+    liveLink: 'https://anita00001.github.io/Portfolio-setup/',
+    buttonTwo: 'See Source',
+    sourceLink: 'https://github.com/anita00001/Portfolio-setup/tree/popup-window',
+  },
+
+  {
+    mTitle: 'Profesional Art Printing Data',
+    dTitle: 'Data Dashboard Healthcare',
+    techList: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    backgroundImg: 'images/modal-bg-2-5.png',
+    p: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    buttonOne: 'See Live',
+    liveLink: 'https://anita00001.github.io/Portfolio-setup/',
+    buttonTwo: 'See Source',
+    sourceLink: 'https://github.com/anita00001/Portfolio-setup/tree/popup-window',
+  },
+
+  {
+    mTitle: 'Profesional Art Printing Data',
+    dTitle: 'Website Protfolio',
+    techList: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    backgroundImg: 'images/modal-bg-3-6.png',
+    p: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    buttonOne: 'See Live',
+    liveLink: 'https://anita00001.github.io/Portfolio-setup/',
+    buttonTwo: 'See Source',
+    sourceLink: 'https://github.com/anita00001/Portfolio-setup/tree/popup-window',
+  },
+];
+
+cardButton.forEach((cardButtonItem, index) => {
+  const items = projectInfo[index];
+  cardButtonItem.addEventListener('click', () => {
+    navBarSection.classList.toggle('active');
+    headlineSection.classList.toggle('active');
+    workSection.classList.toggle('active');
+    aboutMeSection.classList.toggle('active');
+    contactFormSection.classList.toggle('active');
+    socialMediaSection.classList.toggle('active');
+
+    const modalWindow = document.querySelector('body');
+    const section = document.createElement('section');
+    modalWindow.appendChild(section);
+    section.innerHTML = `<div class="modal" id='modal'>
+      <div class="modal-header">
+        <h2 class="modal-mobile">${items.mTitle}</h2>
+        <h2 class="modal-desktop">${items.dTitle}</h2>
+        <button class="close-button">&times;</button>
+      </div>
+      <div class="tech-list">
+        <span class="tech-list-span">${items.techList[0]}</span>
+        <span class="tech-list-span">${items.techList[1]}</span>
+        <span class="tech-list-span">${items.techList[2]}</span>
+      </div>
+      <div class="modal-body">
+        <img class="bg-img-modal" src="${items.backgroundImg}" alt="Featuring image">
+        <section class="modal-description">
+          <p>${items.p}</p>
+          <div class="see-modal-button">
+            <a href="${items.liveLink}"><button class="see-project">${items.buttonOne}<i class="fa-regular fa-arrow-right"></i></button></a>
```

Comment:
- [ ] Good job positioning this see project button :clap: , however the icon within the button is a broken link :broken_heart: , Kindly make sure you correct that to make it professional :call_me_hand:

Reply:
Since the font-awesome icon was not working, we have added the image for the icon (the image was exported from the Figma template in .png format).